tests/flows/validation/test_transfer.py

```diff
@@ -416,9 +417,18 @@ def test_validate_transfer_no_items(
         "adobe_vipm.flows.validation.transfer.get_transfer_by_authorization_membership_or_customer",
         return_value=[],
     )
+    http_error = requests.HTTPError()
+    http_error.response = mocker.MagicMock(
+        spec=requests.Response, wraps=requests.Response
+    )
+    http_error.response.status_code = 400
+    http_error.response.json.return_value = {
+        "rql_validation": ["Value has to be a non empty array."]
+    }
+
     mocker.patch(
         "adobe_vipm.flows.validation.transfer.get_product_items_by_skus",
-        return_value=[],
```

Comment:
I found a test already covering a case of a transfer with no items, so I did not add additional tests. However, I came to conclusion that there was a wrong assumption made in preparation phase of this test. In reality the mock of `get_product_items_by_skus` library function should throw a HTTPError, in case of validation of an order with no items, not just return an empty list, as it is what happened in reality in such a scenario. After implementing the fix it will not happen anyway, but it reflects real scenarios better and will safeguard against future modifications which could introduce a regression around logic implemented here.